libs/@local/hash-graphql-shared/src/graphql/type-defs/knowledge/page.typedef.ts

```diff
@@ -165,29 +165,6 @@ export const pageTypedef = gql`
     ownedById: OwnedById!
   }
 
-  """
-  Create an entity type, which you can then reference in future CreateEntityActions
-  """
-  input CreateEntityTypeAction {
```

Comment:
This action has been made redundant by the type fetcher, so I've removed it from the save action